processing/src/main/java/io/druid/segment/incremental/IncrementalIndexStorageAdapter.java

```diff
@@ -729,6 +768,18 @@ public boolean matches()
         }
       };
     }
+
+    private ValueMatcher makeLongValueMatcher(String dimension, DruidLongPredicate predicate)
+    {
+      // __time is the only long column currently supported
```

Comment:
Should have a sanity check here just in case, something like:

```java
if (!Column.TIME_COLUMN_NAME.equals(dimension)) {
  throw new ISE("Cannot make valueMatcher for column[%s]", dimension);
}
```